lib/stream_data.ex

```diff
@@ -855,7 +883,10 @@ defmodule StreamData do
     filter(enum_data, &not(Enum.empty?(&1)))
   end
 
-  @doc """
+  # QUESTION: Should we change the arguments order of the function
+  # below? So the generator is the first arg and not a function?
```

Comment:
Good point. I actually made the mistake of blindly inspiring this from Clojure's implementation but they invert their arguments compared to what we do, so it totally makes sense.